prusti-viper/src/encoder/mir/pure/specifications/encoder_poly.rs

```diff
@@ -127,6 +127,31 @@ pub(super) fn inline_spec_item<'tcx>(
         .replace_multiple_places(&body_replacements))
 }
 
+fn error_ctxt(resource_type: &vir_crate::polymorphic::ResourceType) -> ErrorCtxt {
```

Comment:
This function specifically returns the error context related to time resources, so either rename it to something that includes `time`, move it inside `encode_time_specifications`, or even remove it (is it only used once).